_posts/aws/2019-06-03-AWS-api-gateway-integration.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2019-05-29
+title: AWS API Gateway Integration
+categories:
+   - AWS
```

Comment:
I think the main category would be `Admin`, since this article is more related with Spinnaker administration. Usually `AWS` category refers to how to make deployments to AWS EC2 instances.